Composer/plugins/runtimes/tsconfig.json

```diff
@@ -5,9 +5,9 @@
     "declaration": true,
     "sourceMap": true,
     "esModuleInterop": true,
-    "skipLibCheck": true,
     "outDir": "./lib",
     "rootDir": "./src",
+    "skipLibCheck": true,
```

Comment:
why was this change necessary?

Reply:
because the packages in node_modules would be checked without this, and may cause type check error.